Rakefile

```diff
@@ -8,7 +8,7 @@ Rake::TestTask.new do |t|
 end
 
 desc 'Run tests'
-task default: [:test, :rubocop]
+task default: %i[test rubocop]
```

Comment:
Could you revert this? We'd prefer to stick to simple non-literal notation for short arrays.